runtime/parachains/src/hrmp.rs

```diff
@@ -492,6 +492,30 @@ pub mod pallet {
 			Ok(())
 		}
 
+		/// Call to establish a channel from the sender to the recipient.
+		///
+		/// This is equivalent to sending an `hrmp_init_open_channel` extrinsic followed by
+		/// `hrmp_accept_open_channel`.
+		///
+		/// Origin must be Root.
+		#[pallet::weight(<T as Config>::WeightInfo::hrmp_init_open_channel() + <T as Config>::WeightInfo::hrmp_accept_open_channel())]
+		pub fn force_establish_hrmp_channel(
+			origin: OriginFor<T>,
+			sender: ParaId,
+			recipient: ParaId,
+			proposed_max_capacity: u32,
+			proposed_max_message_size: u32,
+		) -> DispatchResult {
+			ensure_root(origin)?;
+			Self::init_open_channel(
+				sender,
+				recipient,
+				proposed_max_capacity,
+				proposed_max_message_size,
+			)?;
+			Self::accept_open_channel(recipient, sender)
+		}
```

Comment:
Nit: Can we slap an `Ok(())` at the end here? Just for the consistency sake with the code around

Reply:
This is my fault, I exactly proposed to remove the `Ok(())` :P

Reply:
I'm with Basti on this one.